src/main/osd/feature.h

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Missing licence header and/or too much empty lines.
PS: realy nice fc cant wait to see some pictures :)